packages/expo-maps/ios/AppleMapsViewiOS18.swift

```diff
@@ -1,17 +1,42 @@
 import SwiftUI
 import MapKit
+import ExpoModulesCore
 
 @available(iOS 18.0, *)
 struct AppleMapsViewiOS18: View, AppleMapsViewProtocol {
   @EnvironmentObject var props: AppleMapsViewProps
-  @ObservedObject private var state = AppleMapsViewState()
+  @ObservedObject private var state = AppleMapsViewiOS18State()
 
   func setCameraPosition(config: CameraPosition?) {
     withAnimation {
       state.mapCameraPosition = config.map(convertToMapCamera) ?? .userLocation(fallback: state.mapCameraPosition)
     }
   }
 
+  func openLookAround(coordinate: Coordinate, promise: Promise) {
+    Task {
+      if state.lookAroundScene != nil {
+        promise.reject(LookAroundAlreadyPresentedException())
+        return
+      }
+
+      let scene = await getLookAroundScene(from: CLLocationCoordinate2D(
+        latitude: coordinate.latitude,
+        longitude: coordinate.longitude
+      ))
+
+      if scene == nil {
+        promise.reject(SceneUnavailableAtLocationException())
+        return
+      }
+
+      state.lookAroundScene = scene
+      state.lookAroundPresented = true
+
+      promise.resolve()
+    }
+  }
```

Comment:
```suggestion
  func openLookAround(coordinate: Coordinate) async throws {
      if state.lookAroundScene != nil {
        throw LookAroundAlreadyPresentedException())
      }

      let scene = try await getLookAroundScene(from: CLLocationCoordinate2D(
        latitude: coordinate.latitude,
        longitude: coordinate.longitude
      ))

      state.lookAroundScene = scene
      state.lookAroundPresented = true
  }
```
Same for the other one